source/emit.test.ts

```diff
@@ -0,0 +1,237 @@
+import { expect, test, vi } from "vitest";
+import { emitToString } from "./emit";
+import { QuerySchemasResponse, Schema } from "@ftrack/api";
+import { readFile } from "fs/promises";
+import { join } from "path";
+
+test("emitting with no schemas returns error", async () => {
+  //act
+  const emitResult = await emitToString(
+    "4.13.8",
+    "https://ftrack.example.com",
+    []
+  );
+
+  //assert
+  expect(emitResult?.errors[0]).toBe("No schemas found!");
+});
+
+test("schema subtype of TypedContext", async () => {
+  //arrange
+  vi.setSystemTime(new Date(2023, 1, 1));
```

Comment:
Might be easier to add this to beforeEach

Reply:
Yeah I did consider this, but one of the tests (and possibly more in the future) are not snapshot tests. Let me know if you still think I should add it anyway, and I'll do it 🙏

Reply:
Actually, at second thought, this is just in general a good idea. I did it now.